extensions/reverso-context/package.json

```diff
@@ -13,7 +13,7 @@
   "commands": [
     {
       "name": "context",
-      "title": "Translate with the context",
+      "title": "Translate Word",
```

Comment:
@pernielsentikaer thank you for contributing, but just for clarifying - it actually can translate not only a word but a phrase as well (as you can see in my 4th screenshot), so I guess "Translate Word" could drive users away from using it for phrases.  
Meanwhile, for example, for me, this is the most popular use case to use Reverso Context :)